docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -112,65 +112,48 @@ image::images/timestamp-override.png[]
-events, use the Security:Windows {ml} jobs. If you want alerts for anomalies in
-these jobs, make clones of the existing {ml} rules and update them to use these
-jobs.
-
-If you are cloning prebuilt {ml} rules to generate alerts for the new {ml} jobs,
-the following rules are affected:
-
-* <<unusual-linux-network-port-activity>>: Use
-`v2_linux_anomalous_network_port_activity_ecs` instead of
-`linux_anomalous_network_port_activity_ecs`.
-* <<anomalous-process-for-a-linux-population>>: Use
-`v2_linux_anomalous_process_all_hosts_ecs` instead of
-`linux_anomalous_process_all_hosts_ecs`.
-* <<unusual-linux-username>>: Use `v2_linux_anomalous_user_name_ecs` instead of
-`linux_anomalous_user_name_ecs`.
-* <<unusual-linux-process-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_process` instead of `linux_rare_metadata_process`.
-* <<unusual-linux-user-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_user` instead of `linux_rare_metadata_user`.
-* <<unusual-process-for-a-linux-host>>: Use `v2_rare_process_by_host_linux_ecs`
-instead of `rare_process_by_host_linux_ecs`.
-* <<unusual-process-for-a-windows-host>>: Use
-`v2_rare_process_by_host_windows_ecs` instead of
-`rare_process_by_host_windows_ecs`.
-* <<unusual-windows-network-activity>>: Use
-`v2_windows_anomalous_network_activity_ecs` instead of
-`windows_anomalous_network_activity_ecs`.
-* <<unusual-windows-path-activity>>: Use `v2_windows_anomalous_path_activity_ecs`
-instead of `windows_anomalous_path_activity_ecs`.
-* <<anomalous-windows-process-creation>>: Use
-`v2_windows_anomalous_process_creation` instead of
-`windows_anomalous_process_creation`.
-* <<anomalous-process-for-a-windows-population>>: Use
-`v2_windows_anomalous_process_all_hosts_ecs` instead of
-`windows_anomalous_process_all_hosts_ecs`.
-* <<unusual-windows-username>>: Use `v2_windows_anomalous_user_name_ecs` instead
-of `windows_anomalous_user_name_ecs`.
-* <<unusual-windows-process-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_process` instead of `windows_rare_metadata_process`.
-* <<unusual-windows-user-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_user` instead of `windows_rare_metadata_user`.
+{ml-cap} detection rules use {ml} jobs that have dependencies on data fields populated by the {beats} and {agent} integrations. In version 8.3, new {ml} jobs (prefixed with `v3`) were released to operate on the ECS fields available at that time. 
+
+If you're using 8.2 or earlier versions of {beats} or {agent} with {stack} version 8.3 or later, you may need to duplicate prebuilt rules or create new custom rules _before_ you update the Elastic prebuilt rules. Once you update the prebuilt rules, they will only use `v3` {ml} jobs. Duplicating the relevant prebuilt rules before updating them ensures continued coverage, by allowing you to keep using `v1` or `v2` jobs in the duplicated rules while running alongside the new `v3` jobs in the updated prebuilt rules.
```

Comment:
```suggestion
If you're using 8.2 or earlier versions of {beats} or {agent} with {stack} version 8.3 or later, you may need to duplicate prebuilt rules or create new custom rules _before_ you update the Elastic prebuilt rules. Once you update the prebuilt rules, they will only use `v3` {ml} jobs. Duplicating the relevant prebuilt rules before updating them ensures continued coverage by allowing you to keep using `v1` or `v2` jobs while also running the new `v3` jobs.
```

Reply:
The wording in the last sentence is a little confusing to me. I interpreted it as a recommendation for users to  duplicate relevant V1 and V2 prebuilt rules before upgrading so that they could continue running those duplicate rules alongside the newer V3 rules.

Reply:
Discussed with @nastasha-solomon on Slack: it's important to still specify which rules are running with which ML jobs, so this tries to make that clearer:
```suggestion
If you're using 8.2 or earlier versions of {beats} or {agent} with {stack} version 8.3 or later, you may need to duplicate prebuilt rules or create new custom rules _before_ you update the Elastic prebuilt rules. Once you update the prebuilt rules, they will only use `v3` {ml} jobs. Duplicating the relevant prebuilt rules before updating them ensures continued coverage by allowing you to keep using `v1` or `v2` jobs (in the duplicated rules) while also running the new `v3` jobs (in the updated prebuilt rules).
```